_posts/2024-05-09-djangocon-us-call-for-venue-proposal-2025.md

```diff
@@ -0,0 +1,106 @@
+
+3. A short list of possible sprint venues, with the following information:
+
+	* Name
+	* Location
+	* Transportation options (how do people get there from the hotel?)
+	* Attendance capacity
+	* Information on local companies that might be willing to host the sprints at no cost.
+	* A good sprint venue is/has the following:
+		* Easy to reach from the conference hotel
+		* A room to hold 50-75 people, with work tables, chairs, power, and internet connectivity
+		* Eating tables should be separate from work tables if at all possible.
+		* Excellent Internet connectivity.
+
+4. Hotel. The ideal hotel is adjacent to the conference venue, or a very short walk away. For the hotel. You can reach out to the event sales contact at the hotel to help gather the information we need:
+
+	* Name
+	* Location
+	* Transportation options (how do people get there from the airport?)
+	* The conference room rate (should be discounted from the standard rate)
+	* A block of rooms starting the two nights before tutorials, and running through the last night of the sprints. The breakdown of the room counts would be estimated at:
+	  * Friday - 10
+	  * Saturday - 15
+	  * Sunday Tutorials - 100
+	  * Monday Main Conf - 100
+	  * Tuesday Main Conf - 100
+	  * Wednesday Main Conf - 80
+	  * Thursday Sprints - 30
+	  * Friday Sprints - 15
+
+	* Concessions policy (comp room nights per actualized room nights)
+	* A sample contract showing the terms of booking the block of rooms. Do not sign any contracts. DEFNA will handle the business side of things. We just need to see the terms the hotel proposes for the group block.
+
+5. Provide estimates on costs associated with flights, accommodation, meals, ground transportation, etc. for an overview of the total cost for attendees.
+
+   * Suggested cost examples include typical round-trip airfare from several major hub cities, including New York, Los Angeles, San Francisco, Atlanta, Dallas, Chicago, and London.
+
+6. Information about what makes the city a good choice for DjangoCon US and an attractive destination for attendees.
+
+7. Any additional ideas for making the event especially useful and/or memorable.
+
+8. Any information about whether venues would be willing to sign multi-year deals. Hosting DjangoCon US at the same Marriott in San Diego made venue coordination in 2019 and 2022 considerably easier than having to interact with a new venue each year.
+
+Since we are considering a number of proposals, *no one from your group should sign a binding contract* or make any kind of legal or financial commitment.
+
+If DEFNA chooses your group's proposal, your group is responsible for putting DEFNA in contact with the venue management to finalize the event agreement.
+
+In order for us to make a decision in time to announce next year’s venue at the end of the Durham conference, please submit your proposal no later than July 26, 2024.
```

Comment:
I moved this up to a Tuesday rather than a Sunday. I think it's fair if we get this out in the next two weeks.

Reply:
July 26 is a Friday, which maybe is what you meant. I think I'd be inclined to make it the 28th so as to give folks who are doing this proposal some weekend days to finish it off. Not like we'd be spending a lot of time reviewing it in the 2 days after submission and need the weekend ourselves. Not a strong preference, though. Happy either way.

Reply:
I think the dates are arbitrary, but both/either make sense to me.